.golangci.yaml

```diff
@@ -12,6 +12,16 @@ linters:
     # FIXME: for some reason this errors on windows
     # - gofmt
     # - goimports
+    # TODO: disable for now
+    # - intrange
     - misspell
     - nilerr
+    - nolintlint
+    - nosprintfhostport
+    - testifylint
+    - unconvert
     - unused
+    # TODO: bmizerany says no :(
+    # - usestdlibvars
```

Comment:
```suggestion
```

Let's just leave this undocumented for now

Reply:
moved this to "warn" only